tf_files/configs/kube-setup-networkpolicy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Apply network policy to the core services of the commons
+#
+
+set -e
+
+_KUBE_SETUP_NETWORKPOLICY=$(dirname "${BASH_SOURCE:-$0}")  # $0 supports zsh
+# Jenkins friendly
+export WORKSPACE="${WORKSPACE:-$HOME}"
+export GEN3_HOME="${GEN3_HOME:-$(cd "${_KUBE_SETUP_NETWORKPOLICY}/../.." && pwd)}"
+export XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR:-/tmp}"
+
+if [[ -z "$_KUBES_SH" ]]; then
+  source "$GEN3_HOME/kube/kubes.sh"
+fi # else already sourced this file ...
+
+vpc_name=${vpc_name:-$1}
+if [ -z "${vpc_name}" ]; then
+   echo "Usage: bash kube-setup-networkpolicy.sh vpc_name"
+   exit 1
+fi
+
```

Comment:
Hi Renuka!

Looks great - let's add a check for the kubernetes version:

```
serverVersion=$(kubectl version server -o json | jq -r '.serverVersion.major + "." + .serverVersion.minor + ".0"')
if ! semver_ge "$serverVersion" "1.8.0"; then
  echo "K8s server version $serverVersion does not yet support network policy"
  exit 0
fi
```

Reply:
added with a slight change:
serverVersion="$(kubectl version server -o json | jq -r '.serverVersion.major + "." + .serverVersion.minor' | head -c3).0"
echo "K8s server version is $serverVersion"
if ! semver_ge "$serverVersion" "1.8.0"; then
  echo "K8s server version $serverVersion does not yet support network policy"
  exit 0
fi